src/sdk/pynni/nni/pbt_tuner/pbt_tuner.py

```diff
@@ -0,0 +1,337 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""
+pbt_tuner.py
+"""
+
+import copy
+import logging
+import numpy as np
+import os
+
+import nni
+from nni.tuner import Tuner
+from nni.utils import NodeType, OptimizeMode, extract_scalar_reward
+
+import nni.parameter_expressions as parameter_expressions
+
+logger = logging.getLogger('pbt_tuner_AutoML')
+
+# to be revised
+all_checkpoint_dir = os.getenv('NNI_CHECKPOINT_DIRECTORY')
```

Comment:
this directory is for storing checkpoint of tuner not trials

Reply:
now it is passed to the tuner as a parameter